plugins/generator-1.14.4/forge-1.14.4/procedures/world_data_fluidat.java.ftl

```diff
@@ -0,0 +1 @@
+(world.getFluidState(new BlockPos((int)${input$x},(int)${input$y},(int)${input$z})).getBlockState())
```

Comment:
Is this not the same as the checking block at the given coordinates? Or does this work with flowing fluids and normal getblockstate at blockpos does not?

Reply:
This returns water if the block is waterlogged or always underwater (such as seagrass)

Reply:
Then change

`blockly.block.world_data_fluidat=Get fluid at x: %1 y: %2 z: %3 %4`

to say

`blockly.block.world_data_fluidat=Get fluid as block at x: %1 y: %2 z: %3 %4`

to be more clear about what this block returns